### PR TITLE
Publication of LOOPS book 1 by Stephen Kaisler

### PR DESCRIPTION
**Pull request by pamoroso**
This change lists on the [Documentation](https://interlisp.org/software/using-medley#interlisp-books) page the LOOPS books by Stephen Kaisler and links to the first volume.

### CODE REVIEW

**Review by masinter — APPROVED** [review]
sh0uld this say "was an object extension of Interlisp" instead of "is"??

Somewhere there should be a mention of its role compared to CLOS and CommonLoops?
Possibly somewhere else, but raise that as an issue if you agree.

**Comment by pamoroso** [comment]
I changed to "was an object extension of Interlisp".

**Comment by pamoroso** [comment]
Concerning the role of LOOPS I raised [issue #1850](https://github.com/Interlisp/medley/issues/1850).

File: static/documentation/2024-loops-book-1.pdf.pdf
Comment (stumbo) [inline static/documentation/2024-loops-book-1.pdf.pdf:1]: Rename to `static/documentation/2024-loops-book-1.pdf`  -- remove the extra `.pdf`

**Review by stumbo — CHANGES_REQUESTED** [review]

File: static/documentation/2024-loops-book-1.pdf.pdf
Comment (pamoroso) [reply static/documentation/2024-loops-book-1.pdf.pdf:1]: Oops... Good catch, I didn't realize. Fixed.

**Review by stumbo — APPROVED** [review]

**Pull request merged**